docs/pages/nft/[nftAddress].tsx

```diff
@@ -48,10 +69,162 @@ export default function NftPage({
   }
 
   if (nft === null) {
-    return <div>No NFT</div>;
+    return (
+      <>
+        <SocialHead subtitle="NFT Not Found" />
+        <h1>We couldn’t find an NFToken with this address.</h1>
+        <style jsx>{`
+          h1 {
+            font-size: 1.5rem;
+          }
+        `}</style>
+      </>
+    );
   }
 
-  return <div>NFT {nft.name}</div>;
+  return (
+    <>
+      <SocialHead subtitle={nft.name} />
+
+      <div className="wrapper">
+        <div className="columns">
+          <img src={nft.image} />
+          <div>
+            <h1>{nft.name}</h1>
+
+            <div className="table">
+              {KEYS.map(({ key, type }) => (
+                <React.Fragment key={key}>
+                  <p className="key">{key}</p>
```

Comment:
Let's not use `p` in this component. `<p>` is for blogs / articles. Instead we should use `<div>`